e2e/cypress/plugins/external_request.js

```diff
@@ -34,11 +32,19 @@ module.exports = async ({baseUrl, user, method = 'get', path, data = {}}) => {
             },
             data,
         });
+
+        return {
```

Comment:
Here and in the catch, if we just straight up return the response cypress  complains to me about circular references in the JSON. So just being explicit and returning what we currently care about or need.